deployments.json

```diff
@@ -729,11 +729,6 @@
     "chainId": 314159,
     "url": "https://calibration.filscan.io/en/tx/0xdbfa261cd7d17bb40479a0493ad6c0fee435859e37aae73aa7e803f3122cc465/"
   },
-  {
-    "name": "zKatana",
-    "chainId": 1261120,
-    "url": "https://zkatana.blockscout.com/address/0xcA11bde05977b3631167028862bE2a173976CA11?tab=contract"
-  },
```

Comment:
Do you have a reference for this chain being discontinued, just so I can verify?

Reply:
It's stated in our docs: https://docs.astar.network/docs/learn/networks#zkatana-sepolia